repo/packages/H/helloworld/0/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "helloworld",
+  "version": "0.1.0",
+  "website": "https://github.com/mesosphere/dcos-helloworld",
+  "maintainer": "support@mesosphere.io",
+  "description": "Example DCOS Subcommand",
```

Comment:
This package is more than just a subcommand.  How about "Example DCOS application package"?